src/handlers/Firefox60.ts

```diff
@@ -304,16 +304,20 @@ export class Firefox60 extends HandlerInterface
 
 		let reverseEncodings;
 
-		if (encodings && encodings.length > 1)
+		if (encodings && encodings.length > 0)
 		{
-			encodings.forEach((encoding, idx) =>
+			reverseEncodings = utils.clone(encodings, []);
+			if (encodings.length > 1)
 			{
-				encoding.rid = `r${idx}`;
```

Comment:
Is the mutation of the passed-in `encodings` is intentional?

Reply:
Yes, because if multiple `encodings` are given then `rid` is mandatory. However we do it internally in mediasoup-client rather than requiring the user to do it.